README.md

```diff
@@ -11,19 +13,41 @@ For the Points developer take-home assignment, the following task is required: y
     * Calculates and displays the total taxes owed for the salary
     * Displays the amount of taxes owed per band
     * Displays the effective rate
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+
+### Level 1
+
+* Fetch the tax rates from this server at: [localhost:5000/tax-calculator/brackets](http://localhost:5000/tax-calculator/brackets)
+
+### Level 2
+
+* Fetch the tax rates by year: [localhost:5000/tax-calculator/brackets/<2019,2020...>](http://localhost:5000/tax-calculator/brackets/2020)
 
-There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+_Warning: this endpoint has gotchas_
+
+## Assignment 2: Autonomous Car
+
+### Instructions
+
+* Download and run this server (see [Getting Started](#getting-started))
+* Build an application / tool in React or Python with the following criteria:
+    * Fetches the a autonomous car route from this API server
+    * Takes a set of instructions as input
+    * Determine's if the route was a success or a failure
+    * If it was a failure, determine at which step or position that failed
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
 
 ### Level 1
 
-* Fetch the tax rates from this server at: http://localhost:5000/tax-calculator/brackets
+* Fetch a random autonomous car route from this server at: [localhost:5000/autonomous-car/routes/](http://localhost:5000/autonomous-car/routes/)
 
 ### Level 2
 
-* Fetch the tax rates by year: http://localhost:5000/tax-calculator/brackets/<2019|2020|...>
+* Fetch an autonomous car route by status: [localhost:5000/autonomous-car/routes/<success|failure|empty>](http://localhost:5000/autonomous-car/routes/empty)
```

Comment:
I'm confused about this car route by status. Can you elaborate on its intent?

Reply:
The idea was that the user could choose what type of route they wanted to test their code with.

Success would return routes that the applicant's code should reach the end without hitting any obstacles or going out of bounds.

Failure would return route where the car goes out of bounds or hits an obstacle. Causing a failure.

Empty ... well I guess just another random route they should be able to handle

Reply:
I'm planning on changing this LVL 2 endpoint to be as you recommended in your other comment.

Level 1: Always returns a successful route
Level 2: Returns random routes that could result in success or failure

Reply:
After meeting up: 
```
Level 1 - /autonomous-car/routes/ => Static (successful) car route 
Level 2 - /autonomous-car/routes/random => random car route with gotchas  
```